From 72bda7188408bd3c2491ea664da294cc483220bc Mon Sep 17 00:00:00 2001
From: Andrey Rakhmatullin <wrar@wrar.name>
Date: Sun, 20 Nov 2016 18:40:49 +0500
Subject: [PATCH] Set CMP0054 CMake policy to NEW.

CMake 3.7 exports a "t" variable for all projects and because of that
the elseif statement in cmake/CMakeCSharpInformation.cmake:311 works
incorrectly if CMP0054 is not set to NEW (as "t" is expanded to the
variable value).
---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5313787..451e7a6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,6 +3,9 @@ cmake_minimum_required(VERSION 2.6)
 if(POLICY CMP0011)
    cmake_policy(SET CMP0011 NEW)
 endif(POLICY CMP0011)
+if(POLICY CMP0054)
+   cmake_policy(SET CMP0054 NEW)
+endif(POLICY CMP0054)
 
 # Make sure we look in our cmake folder for additional definitions
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake )
